### PR TITLE
A4A > Agency Tier: Implement benefits sections

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1284

Design: GSGFOUvQQRQbi3rdxEesDB-fi-8577_9820

Note: The top section will be implemented in a different PR.

## Proposed Changes

This PR implements the benefits sections for agency tiers.

## Why are these changes being made?

* To showcase the agency tier benefits. 

## Testing Instructions

- Patch D163771-code if not merged already.
- Open the A4A live link
- Go to Agency Tier > Verify the screen looks good on all the different sizes as shown below.

screencapture-agencies-localhost-3000-agency-tier-2024-10-15-12_22_05

Screenshot 2024-10-15 at 12 04 32 PM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by cleacos — APPROVED** [review]
**A minor comment:** The badge background color doesn't match with the design. And the padding seems different top vs bottom:

image

Other than that: LGTM

**Comment by yashwin** [comment]
Thanks for the review, @cleacos!

I have merged this PR, and will fix the padding issue on another PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16910597

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

File: client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx [hunk 1]
```diff
@@ -1,7 +1,99 @@
...
+			className="agency-tier-overview"
+			title={ title }
+			wide
+			sidebarNavigation={ <MobileSidebarNavigation /> }
+		>
+			<LayoutTop>
+				<LayoutHeader>
+					<Title>{ title } </Title>
+					<Subtitle>
+						{ translate( 'Experience the rewards of selling Auatomattic products and hosting.' ) }
```
Comment (emilyaudela) [inline client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:34]: Typo: Auatomattic --> Automattic

Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:34]: Thanks for reporting! I'll fix this in my upcoming PR.

Comment (jornp) [reply client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:34]: @yashwin This was [fixed](https://github.com/Automattic/wp-calypso/pull/95423) by @sdnunca a couple of hours ago. 👍

Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:34]: Thanks, everyone!

File: client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		isComingSoon: true,
+		availableTiers: [ 'emerging-partner', 'agency-partner', 'pro-agency-partner' ],
+	},
+	{
+		title: translate( 'Tools & Platforms' ),
+		description: translate(
+			'Agency dash for streamlined client billing, program and product management, and free agency site on WordPress.com or Pressable*.'
+		),
+		features: [
+			translate( 'Agency partners receive free WordPress.com and Pressable site.' ),
```
Comment (emilyaudela) [inline client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts:69]: Should the word "site" be plural here?

File: client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx [hunk 3]
```diff
@@ -1,7 +1,99 @@
...
+				</LayoutHeader>
+			</LayoutTop>
+
+			<LayoutBody>
+				<div className="agency-tier-overview__bottom-content">
+					<div className="agency-tier-overview__bottom-content-subheading">
+						{ translate( 'Take a closer look' ) }
+					</div>
+					<div className="agency-tier-overview__bottom-content-heading">
+						{ translate( 'Explore the benefits of the tiers' ) }
```
Comment (emilyaudela) [inline client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:51]: From the screenshot, it looks like the section refers to a single tier. Should this be "Explore the benefits of the tier"? (Questions from the translation team)

Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx:51]: Thanks for checking!

No, this is a section for all the tiers. The three icons on top indicate the tiers. We will have sections at the top that mention what tier each icon is for.

File: client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts (hunk 2 above)
Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts:69]: That's a good catch. I think it's better to keep this `Agency partners receive a free WordPress.com site and a Pressable site.` Let me confirm this with the design team and fix it. Thanks!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16910597).

File: client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts (hunk 2 above)
Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts:69]: I have updated the text here: https://github.com/Automattic/wp-calypso/pull/95548

Thanks!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16910597).

**Pull request merged**